docs/02-guides/adding-your-own-server-timings.mdx

```diff
@@ -0,0 +1,103 @@
+---
+title: Add your own server timings
+description:
+  This guide explains how you can add `Server-Timing` headers in your responses
+  in Front-Commerce.
```

Comment:
Not sure if this is accurate? `Server-Timing` headers will always be there?, instead this guide explains how to add new timings in the existing `Server-Timing` header?